requirements.txt

```diff
@@ -0,0 +1,2 @@
+up-python==0.1.2.dev0
```

Comment:
As per the SemVer standard we use, it would be 0.1.2-dev

Reply:
Updated